### PR TITLE
Assignment1 - Code Review Changes

### PR DESCRIPTION
**Pull request by megha994**
Changes made as per code review comments

### CODE REVIEW

File: _pages/8.html [hunk 1]
```diff
@@ -0,0 +1,398 @@
...
+                <div class="bottom-section-spacer-sm"></div>
+                <p>
+                  Learn about the actions and measures for Community Well-Being
+                  by visiting
+                  <a href=""
+                    ><span class="text-color">
+                      page 10 in the 2023-2026 Strategtic Business Plan</span
+                    ></a
+                  >
+                </p>
```
Comment (ericnjanga) [inline _pages/8.html]: Please move this paragraph outside of the list. The designer made an alignment mistake. In fact, the paragraph should be on the same vertical line as the list. Please do the same for the other 3 boxes' paragraphs.

File: _pages/8.html [hunk 2]
```diff
@@ -0,0 +1,398 @@
...
+              </ul>
+            </div>
+          </div>
+        </div>
+
+        <!--------------->
+        <div class="col-md-12 col-sm-12 bx-container">
+          <div class="bottom-section-spacer-sm">
+            <h2 id="01" class="bottom-section-spacer-sm">
```
Comment (ericnjanga) [inline _pages/8.html]: Please remove the class "bottom-section-spacer-sm" here, The h2's natural bottom spacing is enough.

Note: Once again, it's not your fault, the designer made a little exaggeration.

File: _pages/8.html [hunk 3]
```diff
@@ -0,0 +1,398 @@
...
+                    <!------------------------?????---------------------------------->
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+
+          <div class="bottom-section-spacer-sm">
+            <h2 id="01" class="bottom-section-spacer-sm">
+              <span class="modified-heading">
```
Comment (ericnjanga) [inline _pages/8.html]: Please remove the class "bottom-section-spacer-sm" here, The h2's natural bottom spacing is enough.

Note: Once again, it's not your fault, the designer made a little exaggeration.

File: _pages/8.html [hunk 4]
```diff
@@ -0,0 +1,398 @@
...
+        </div>
+        <div class="col-md-5 col-sm-5 bx-container">
+          <div class="img-border bottom-section-spacer-sm">
+            <img src="./../img/image 8.jpeg" class="img-fluid" alt="image 8" />
+          </div>
+        </div>
+        <div class="col-md-7 col-sm-7 bx-container">
+          <div id="1" class="bottom-section-spacer-sm">
+            <h2>
+              <span class="modified-heading"
```
Comment (ericnjanga) [inline _pages/8.html]: I'm not sure why you created this class. Once, I understand more about your thoughts, I'll let you know.

**Review by ericnjanga — CHANGES_REQUESTED** [review]
Please address my comments!

File: _pages/8.html (hunk 1 above)
Comment (megha994) [reply _pages/8.html]: fixed

File: _pages/8.html (hunk 2 above)
Comment (megha994) [reply _pages/8.html]: fixed

File: _pages/8.html (hunk 3 above)
Comment (megha994) [reply _pages/8.html]: fixed

File: _pages/8.html (hunk 4 above)
Comment (megha994) [reply _pages/8.html]: oke

**Pull request merged**